src/modules/fw_rate_control/fw_rate_control_params.c

```diff
@@ -295,6 +295,21 @@ PARAM_DEFINE_INT32(FW_BAT_SCALE_EN, 0);
  */
 PARAM_DEFINE_INT32(FW_ARSP_SCALE_EN, 1);
 
+/**
```

Comment:
@bresch

So to be clear, 
1. initially the TRIM_ROLL/PITCH/YAW are set to 0 (default) and this value is set to 1 (calibration).
2. You take off and fly for a bit (large circles, bank angle < 10degrees) or straight lines for several periods of at least 7 seconds) and land
3. Calibration values are saved to the TRIM_ values, FW_ATRIM_MODE is set to 2 (continuous).

When you fly again, the values of TRIM_ROLL/PITCH/YAW are updated continuously, off the small delta of the saved calibration data:
- FMI Why do the values for trim change?
- Are the modified continuous values written back to the parameters?

Reply:
> So to be clear,
initially the TRIM_ROLL/PITCH/YAW are set to 0 (default) and this value is set to 1 (calibration).
You take off and fly for a bit (large circles, bank angle < 10degrees) or straight lines for several periods of at least 7 seconds) and land
Calibration values are saved to the TRIM_ values, FW_ATRIM_MODE is set to 2 (continuous).

Correct.

> When you fly again, the values of TRIM_ROLL/PITCH/YAW are updated continuously, off the small delta of the saved calibration data:
FMI Why do the values for trim change?

The trim can change due do a different weight distribution: might have a different payload or battery placement.
It could also change after servicing (e.g.: when changing a servo, the control surface might be in a slightly different position than before).

> Are the modified continuous values written back to the parameters?

Yes, but the change is limited (max difference is 0.05).